src/dataTables.altEditor.free.js

```diff
@@ -185,39 +185,44 @@
                 if (dt.button('edit:name')) {
                     dt.button('edit:name').action(function (e, dt, node, config) {
                         that._openEditModal();
-                    });
 
-                    $(this.modal_selector).on('submit', '#altEditor-edit-form', function (e) {
-                        console.log("EDDDIT");
-                        e.preventDefault();
-                        e.stopPropagation();
-                        that._editRowData();
+                        $('#altEditor-edit-form')
```

Comment:
What if there are two datatables in the same HTML page? can they be in conflict?

Reply:
Fixed.